docker-compose.yml

```diff
@@ -23,7 +23,9 @@ services:
     environment:
       # Adapted from ./config/config.yml
       INVIDIOUS_CONFIG: |
```

Comment:
Please don't change those, this will conflict with https://github.com/iv-org/invidious/pull/2917

Reply:
Okay, I'll clean it up right away